### ISSUE TITLE
"Nothing to do" for `./letsencrypt-auto certonly`

### ISSUE THREAD

**Issue by yiding-he** [certbot/certbot#2971]
when I run `./letsencrypt-auto certonly` it is not generating cert files, instead it only shows "Nothing to do". I don't know why.

```
[root@iZ948375vfsZ letsencrypt]# hostnamectl
   Static hostname: iZ948375vfsZ
         Icon name: computer-vm
           Chassis: vm
        Machine ID: 45461f76679f48ee96e95da6cc798cc8
           Boot ID: bf4fac63fbf94908968d42b32f97b31e
    Virtualization: xen
  Operating System: CentOS Linux 7 (Core)
       CPE OS Name: cpe:/o:centos:centos:7
            Kernel: Linux 3.10.0-123.9.3.el7.x86_64
      Architecture: x86-64

[root@iZ948375vfsZ letsencrypt]# ./letsencrypt-auto certonly
Bootstrapping dependencies for RedHat-based OSes...
yum is /usr/bin/yum
Loaded plugins: langpacks
Package python-2.7.5-34.el7.x86_64 already installed and latest version
Package python-devel-2.7.5-34.el7.x86_64 already installed and latest version
Package python-virtualenv-1.10.1-2.el7.noarch already installed and latest version
Package python-tools-2.7.5-34.el7.x86_64 already installed and latest version
Package python-pip-7.1.0-1.el7.noarch already installed and latest version
Nothing to do
Loaded plugins: langpacks
Package gcc-4.8.5-4.el7.x86_64 already installed and latest version
Package dialog-1.2-4.20130523.el7.x86_64 already installed and latest version
Package augeas-libs-1.4.0-2.el7.x86_64 already installed and latest version
Package 1:openssl-1.0.1e-51.el7_2.5.x86_64 already installed and latest version
Package 1:openssl-devel-1.0.1e-51.el7_2.5.x86_64 already installed and latest version
Package libffi-devel-3.0.13-16.el7.x86_64 already installed and latest version
Package redhat-rpm-config-9.1.0-68.el7.centos.noarch already installed and latest version
Package ca-certificates-2015.2.6-70.1.el7_2.noarch already installed and latest version
Nothing to do
Checking for new version...

[root@iZ948375vfsZ letsencrypt]# python --version
Python 2.7.5

[root@iZ948375vfsZ letsencrypt]#
```

**Comment by yiding-he** [comment]
OK this is fixed after I did following:
1. run `#yum update`;
2. log off;
3. log in again.

then `./letsencrypt-auto certonly` executes correctly.

**Comment by pde** [comment]
Hmmm, this looks like a possible mishandling of a network error while trying to check for a new version of `letsencrypt-auto`.  It should probably have been fixed in Certbot 0.6.0, which was released today.  If you ever see this again, please file a new ticket.
